### PR TITLE
Example of how to configure MQTT agent for mTLS

### PR DESCRIPTION
**Pull request by pankalog**
To be merged after https://github.com/openremote/openremote/pull/1351 is completed.

@DonWillems Let me know if this looks good for you.

### CODE REVIEW

File: docs/user-guide/agents-protocols/mqtt.md [hunk 1]
```diff
@@ -4,11 +4,58 @@ sidebar_position: 7
...
+
+After creating a Thing, you need to retrieve its certificate, its private key, and Amazon's Root CA certificate.
+
+They are all provided after accessing that thing's dashboard and creating a new certificate for it: ![img.png](img/aws-iot-mqtt-broker-download-links.png)
+
+Download the Device Certificate and activate it, download the private key file, and the Amazon Trust Services endpoint RSA 2048 bit key.
+
+Make sure that OpenRemote has been started at least once before proceeding, so that the required keystore files are generated.
+
+The password of everything keystore-related is ``OR_ADMIN_PASSWORD``, for when it is requested.
```
Comment (wborn) [inline docs/user-guide/agents-protocols/mqtt.md]:
```suggestion
The password of everything keystore-related is `OR_ADMIN_PASSWORD`, for when it is requested.
```

**Comment by richturner** [comment]
@pankalog is this ready to be reviewed?

**Comment by richturner** [comment]
@DonWillems not sure this should have been merged, I cannot see mention of environment variables and paths for mapping the keystore(s) into the manager container.

@pankalog was going to confirm.

**Comment by pankalog** [comment]
@richturner @DonWillems no, this wasn't ready 😅

I'll open up a new PR with the updated docs for it when I have it ready, I'll try for it to not take that much time.

**Comment by wborn** [comment]
> no, this wasn't ready

You can create a "Draft" PR to prevent changes from being merged:

image

**Pull request merged**